Pod/Classes/LocationObserver.swift

```diff
@@ -24,38 +24,47 @@ class LocationObserverItem: NSObject {
     @objc var newLocationForUpdate: CLLocation?
     @objc var minimumTimer: Timer?
     @objc var maximumTimer: Timer?
-    
-    init(locationObserver: LocationObserver, locationManager: LocationManager, desiredAccuracy: CLLocationAccuracy?, distanceFilter: CLLocationDistance?, minimumTimeInterval: TimeInterval?, maximumTimeInterval: TimeInterval?) {
+
+    init(locationObserver: LocationObserver,
```

Comment:
I am not happy with this indentation since the body of the moethod is not clearly divided from the signature of the method.

```swift
init(
    locationObserver: LocationObserver,
    locationManager: LocationManager,
    desiredAccuracy: CLLocationAccuracy?,
    distanceFilter: CLLocationDistance?,
    minimumTimeInterval: TimeInterval?,
    maximumTimeInterval: TimeInterval?
) {
    observer = locationObserver
    self.locationManager = locationManager
    self.desiredAccuracy = desiredAccuracy
    self.distanceFilter = distanceFilter
    self.minimumTimeInterval = minimumTimeInterval
    self.maximumTimeInterval = maximumTimeInterval

    super.init()
    initializeTimers()
}
```

Reply:
Hello @mkj-is,
I changed the formatting of code in this pull request, please check if it is OK now.